.github/workflows/asv_check.yml

```diff
@@ -0,0 +1,34 @@
+name: asv
+
+# CI ASV CHECK is aimed to verify that the benchmarks execute without error.
+on: [pull_request, push]
```

Comment:
@kanderso-nrel do we really need this on push? I was surprised to push a branch to my fork and then get a notification that the test was done.

Reply:
I thought it would be nice to run it on pushes to master. I didn't consider the effect on forks.  Fine with me to change it to something like this:
```yaml
on:
  push:
    branches:
    - master
  pull_request:
```